src/goto-symex/equation_conversion_exceptions.h

```diff
@@ -1,47 +0,0 @@
-/*******************************************************************\
```

Comment:
I don't see why this was removed! (see also my comments where this exception class was moved to....)